doc/performance-investigations/README.md

```diff
@@ -0,0 +1,29 @@
+# Performance Investigations
+
+## Creating a new investigation
+
+1. Create a branch in runner prefixed with `performance-investigation` and make the proposed changes.
+1. In this repo create a new folder named `0000-descriptive-name-for-investigation` incrementing `0000` from the last used number.
+1. Copy `TEMPLATE.md` into the directory as `summary.md`.
+1. Update `summary.md` as appropriate.
+1. Capture baseline metrics - `./run.sh requests/<requests_file> https://<runner_url> <duration> baseline`.
```

Comment:
In the summary.md we are looking to record the user wait time minimum; user wait time maximum; clients and hatch rate. But it's not obvious what they are set to, or how we change them from this readme. A more explicit command would help possibly

Reply:
Readme expanded.